next.config.js

```diff
@@ -1,7 +1,8 @@
 /** @type {import('next').NextConfig} */
 const nextConfig = {
   reactStrictMode: true,
-  swcMinify: true
+  swcMinify: true,
+  images: { domains: ['prototyprio.gumlet.io'] }
```

Comment:
The recommended domain addition method in configuration with Next 13 should be as follows. **[Remote Patterns](https://beta.nextjs.org/docs/optimizing/images#defining-domains-for-remote-images)**